apps/wallet/src/ui/app/components/alert/index.tsx

```diff
@@ -5,20 +5,31 @@ import cl from 'classnames';
 import { memo } from 'react';
 
 import Icon, { SuiIcons } from '_components/icon';
+import LoadingIndicator from '_components/loading/LoadingIndicator';
 
 import type { ReactNode } from 'react';
 
 import st from './Alert.module.scss';
 
+type ModeType = 'warning' | 'loading' | 'success';
 export type AlertProps = {
     children: ReactNode | ReactNode[];
     className?: string;
+    mode?: ModeType;
+};
+const modeToIcon: Record<Exclude<ModeType, 'loading'>, SuiIcons> = {
+    warning: SuiIcons.Info,
+    success: SuiIcons.Check,
 };
 
-function Alert({ children, className }: AlertProps) {
+function Alert({ children, className, mode = 'warning' }: AlertProps) {
     return (
-        <div className={cl(st.container, st.error, className)}>
-            <Icon className={st.icon} icon={SuiIcons.Info} />
+        <div className={cl(st.container, st[mode], className)}>
+            {mode === 'loading' ? (
+                <LoadingIndicator className={st.loader} />
```

Comment:
In general I think we should avoid passing custom classnames through like this and instead make the indicator driven through props so that it can have a fixed set of possible styles.